python/src/trezorlib/cli/benfen.py

```
@@ -0,0 +1,60 @@
+@with_client
+def get_address(client: "TrezorClient", address: str, show_display: bool) -> str:
+    """Get Benfen address in hex encoding."""
+    address_n = tools.parse_path(address)
+    return benfen.get_address(client, address_n, show_display)
+
+
+@cli.command()
+@click.option("-n", "--address", required=True, help=PATH_HELP)
+@click.argument("message")
+@with_client
+def sign_raw_tx(client: "TrezorClient", address: str, message: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `sign_raw_tx`

(ANN201)